src/components/screens/layout-screen-html.tsx

```diff
@@ -196,8 +207,7 @@ const LayoutScreenHTML: FC<Props> = ({
     },
     {
       name: strings.exhibitionLayouts.editView.saveButton,
-      action: onLayoutSave,
-      disabled: !dataChanged
```

Comment:
Why data changed functionality has been removed?

Reply:
Added it back per discussions